BackofficeBundle/Manager/NodeManager.php

```diff
@@ -92,6 +97,20 @@ public function createNewLanguageNode(NodeInterface $node, $language)
     }
 
     /**
+     * @param NodeInterface $node
+     *
+     * @return StatusInterface
+     */
+    protected function getInitialStatus(NodeInterface $node)
```

Comment:
I haven't seen it yet but this method is not consistent.

You cannot say getInitialStatus and then return only an editable status.